receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -322,15 +366,14 @@ func (jr *jReceiver) EmitZipkinBatch(spans []*zipkincore.Span) error {
 // EmitBatch implements cmd/agent/reporter.Reporter and it forwards
 // Jaeger spans received by the Jaeger agent processor.
 func (jr *jReceiver) EmitBatch(batch *jaeger.Batch) error {
-	td, err := jaegertranslator.ThriftBatchToOCProto(batch)
+	_, span := obsreport.StartTraceDataReceiveOp(
```

Comment:
Should this use the same `consumeTraceData` method above?

Reply:
This one is not a slice but a single batch.